tasks/queue-granules/src/index.ts

```diff
@@ -174,6 +174,7 @@ async function queueGranules(event: HandlerEvent): Promise<QueueGranulesOutput>
                 body: {
                   collectionId,
                   granuleId,
+                  producerGranuleId: producerGranuleId || granuleId,
```

Comment:
We talked about this, we need to proactively address PATCH requiring anything other than granuleId/collectionId.... and probably granuleId.     I was unable to find a ticket, so issued CUMULUS-4093.  This is fine for this PR, but we should consider fixing this under the decom issue if we have time.